src/xenia/kernel/xam/xam_content.cc

```diff
@@ -49,7 +49,7 @@ dword_result_t XamContentGetLicenseMask(lpdword_t mask_ptr,
   // Each bit in the mask represents a granted license. Available licenses
   // seems to vary from game to game, but most appear to use bit 0 to indicate
   // if the game is purchased or not.
-  *mask_ptr = 0;
+  *mask_ptr = (cvars::license_mask);
```

Comment:
Needs a cast. `static_cast<uint32_t>(cvars::license_mask)` I think should work?

Reply:
Resolved.